deps/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+project(ziti-tunneler-sdk-deps)
+
+include(ExternalProject)
+
+add_subdirectory(ziti-sdk-c)
```

Comment:
no newline at end of file.   lwip and lwip-contrib?

Reply:
lwip and lwip-contrib are added in a more indirect way - by including lwip/src/Filelists.cmake.